pkg/morph/event/listener.go

```diff
@@ -272,7 +272,6 @@ loop:
 				continue loop
 			}
 
-			// TODO: consider asynchronous execution
```

Comment:
think it is important

Reply:
It can be discussed, indeed. But I propose to keep block handlers sync:
1. there is one block handler per application now
2. there is no performance issue with sync execution, we can consider that when we will need extra performance on that.